flexmeasures/api/v3_0/sensors.py

```diff
@@ -470,20 +481,23 @@ def get_schedule(  # noqa: C901
 
             fallback_job_id = job.meta.get("fallback_job_id")
 
-            # redirect to the fallback schedule endpoint if the fallback_job_id
-            # is defined in the metadata of the original job
-            if fallback_job_id is not None:
-                return fallback_schedule_redirect(
-                    message,
-                    url_for(
-                        "SensorAPI:get_schedule",
-                        uuid=fallback_job_id,
-                        id=sensor.id,
-                        _external=True,
-                    ),
-                )
-            else:
-                return unknown_schedule(message)
+            if current_app.config.get("FLEXMEASURES_FALLBACK_REDIRECT"):
+                # redirect to the fallback schedule endpoint if the fallback_job_id
+                # is defined in the metadata of the original job
+                if fallback_job_id is not None:
+                    return fallback_schedule_redirect(
+                        message,
+                        url_for(
+                            "SensorAPI:get_schedule",
+                            uuid=fallback_job_id,
+                            id=sensor.id,
+                            _external=True,
+                        ),
+                    )
+                else:
+                    return unknown_schedule(message)
+            # case that the job failed or the fallback schedule job failed (FLEXMEASURES_FALLBACK_REDIRECT = False)
+            return unknown_schedule(message)
```

Comment:
I think these changes are redundant. If the original job failed and
- if we are not redirecting, the `job` variable will already point to the fallback job, whose `fallback_job_id is None`, so we `return unknown_schedule(message)`.
- if we *are* redirecting, the `job` variable will still point to the original job, whose `fallback_job_id is not None`, so we `return fallback_schedule_redirect(...)`.

Reply:
Yes, you're right! That will simplify the code.